plugins/ruby-foreman-azure/debian/changelog

```diff
@@ -16,6 +22,12 @@ ruby-foreman-azure (1.1.1-1) stable; urgency=low
 
  -- Daniel Lobato Garcia <me@daniellobato.me>  Tue, 15 Nov 2016 11:32:48 +0100
 
+ruby-foreman-azure (1.1.0-1) stable; urgency=low
```

Comment:
Not sure this should be added, this build didn't ever exist as far as I can tell. Where is it from?